test/org/sagebionetworks/bridge/play/controllers/ApplicationControllerMockTest.java

```diff
@@ -87,18 +87,22 @@ public void before() {
     }
     
     @Test
-    public void verifyEmailWorks() {
+    public void verifyEmailWorks() throws Exception {
         Result result = controller.verifyEmail("test-study");
         
+        assertEquals("text/html; charset=utf-8", result.header("Content-Type"));
+        assertEquals(200, result.status());
```

Comment:
Minor: This should use TestUtils.assertStatus(). (Here and in resetPasswordWorks().)

Reply:
I don't see this method in TestUtils...

Reply:
My bad. This one looks like it's actually different. Disregard.